### DISCUSSION TITLE
Qualifying Brazil

### DISCUSSION THREAD

**Discussion by ChemiF1** [theOehrly/Fast-F1#651] [General]
Hello there

Is anyone able to download the data from the Qualy?

It looks like there's nothing on the replay section from Live Timing App

**Comment by ChemiF1** [comment]
This is explained on issue #652

Data is recorded but not able to load and use because of date issue

As soon our magicians make the change data will be ready

**Reply by theOehrly** [reply]
Done 🧙‍♂️
